src/dbnode/storage/index.go

```diff
@@ -1136,14 +1188,27 @@ func (i *nsIndex) query(
 }
 
 func (i *nsIndex) execBlockQueryFn(
+	ctx context.Context,
 	cancellable *resource.CancellableLifetime,
 	block index.Block,
 	query index.Query,
 	opts index.QueryOptions,
 	state *asyncQueryExecState,
 	results index.BaseResults,
 ) {
-	blockExhaustive, err := block.Query(cancellable, query, opts, results)
+	ctx, sp := ctx.StartTraceSpan(tracepoint.NSIdxBlockQuery)
+	sp.LogFields(
+		opentracinglog.String("query", query.String()),
+		opentracinglog.String("namespace", i.nsMetadata.ID().String()),
+		opentracinglog.Int("limit", opts.Limit),
+		xopentracing.Time("query_start", opts.StartInclusive),
+		xopentracing.Time("query_end", opts.EndExclusive),
+		xopentracing.Time("block_start", block.StartTime()),
+		xopentracing.Time("block_end", block.EndTime()),
```

Comment:
nit: Can we be consistent and use lower camel case for naming log fields and trace log fields? i.e. `queryStart` vs `query_start`.